packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -199,17 +206,43 @@ export function getDefaultProvider(input: FieldProvider): DefaultProvider {
 	return input as unknown as DefaultProvider;
 }
 
+/**
+ * Default field schema metadata shape.
+ *
+ * @remarks
+ *
+ * All properties are strictly optional.
+ * These particular properties exist both as an example of what kinds of data can be associated with a field,
+ * and may also be used by the system and related tooling in some cases.
+ *
+ * @sealed
```

Comment:
If this is sealed, then its invalid for a customer to create an instance of it.

The whole point of this interface is for customers to provide instances of it correct?

Sealed is for types the package produces instances of, and customers must not. We do not have a tag for this input case.


Note if we plan to add more built in members to this in the future, that would be a breaking change due to name collisions with custom app data unless we ban (or somehow scope) app customizations to this. Maybe providing a "custom" sub object would make sense?

Reply:
`@sealed` indicates that the user should not _extend_ the interface.

https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#sealed

I don't think we have a tag that indicates a user should not create an object of a given type. `@system` would probably be the closest option, but still not an exact match.

Reply:
The key collision case is an interesting one, though. `custom` is probably a good option. I'll update to use that pattern.